docs/book/src/tasks/kubeadm-control-plane.md

```diff
@@ -0,0 +1,28 @@
+# Kubeadm control plane
+
+Using the Kubeadm control plane type to manage a control plane provides several ways to upgrade control plane machines.
+
+## Upgrading
+
+The high level steps to fully upgrading a workload cluster are to first upgrade the control plane and then upgrade
+the worker machines.
+
+### Upgrading workload machines managed by a `MachineDeployment`
+
+It is recommended to manage workload machines with a `MachineDeployment` as this will transparently manage `MachineSet`s
+and `Machine`s to allow for a seamless scaling experience. A modification to the `MachineDeployment`s spec will begin a
+rolling update of the workload machines.
+
+### How to upgrade the underlying machine image
```

Comment:
What about grouping this paragraph and the following under a section:
Upgrading control-plane machines managed by a `KubeadmControlPlane` object